database/handlers/parentOrg.go

```diff
@@ -63,22 +63,30 @@ func (client Client) GetAllOrgNames() []string {
  * Args:        name, logo
  * Returns:     org (ParentOrg object with the data inserted)
  */
-func (client Client) CreateParentOrg(name string, logo string) *database.ParentOrg {
+func (client Client) CreateParentOrg(name string, logo string, descriptions string, skills []string, term []string, website string) *database.ParentOrg {
 	// Create a placeholder object.
-	org := database.ParentOrg{ID: "0", Name: name, Logo: logo}
+	org := database.ParentOrg{
+		ID: "0",
+		Name: name, 
+		Logo: logo,
+		Description:  descriptions,
+        Skills:       skills,
```

Comment:
Proper indentation is missing. Also need to update the logic as described below.